packages/video_player/video_player/ios/Classes/FLTVideoPlayerPlugin.m

```diff
@@ -511,7 +511,7 @@ - (FLTTextureMessage*)onPlayerSetup:(FLTVideoPlayer*)player
   return result;
 }
 
-- (void)initialize:(FlutterError* __autoreleasing*)error {
+- (void)initialize:(FlutterError* _Nullable __autoreleasing*)error {
```

Comment:
Use `nullable` in ObjC method signatures.

Reply:
Also, why is this method named incorrectly? This isn't Pigeon-generated presumably.

Reply:
I think you have to use _Nullable since we are passing in a handle to a FlutterError.  It's not named incorrectly, I've used `@ObjcSelector` to make all the selectors backwards compatible with how the code previously worked.